<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1461</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLCDNumber" name="accuracyLCD_2">
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="wpmLCD_2"/>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="testLabel_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="accuracyLabel_2">
       <property name="text">
        <string>Точность, %:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLCDNumber" name="timerLCD_2"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="wpmLabel_2">
       <property name="text">
        <string>Слов в минуту:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="timerLabel_2">
       <property name="text">
        <string>Время, дс:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="startButton_2">
       <property name="text">
        <string>Начать</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="stopButton_2">
       <property name="text">
        <string>Закончить</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="languageChangeComboBox_2">
       <item>
        <property name="text">
         <string>Французский</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Немецкий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Арабский</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Белорусский</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Иврит</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Китайский</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="chooseLanguageLabel_2">
       <property name="text">
        <string>Выбор языка:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="openTestButton_2">
       <property name="text">
        <string>Открыть задание</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
